src/components/AccountForm.vue

```diff
@@ -529,6 +606,8 @@ export default {
 						this.feedback = t('mail', 'IMAP connection failed')
 					} else if (error.data?.service === 'SMTP') {
 						this.feedback = t('mail', 'SMTP connection failed')
+					} else if (error.message === CONSENT_ABORTED) {
+						this.feedback = t('mail', 'Google authorization popup closed')
```

Comment:
@miaulalala is that the message you see?

Reply:
That could mean your popup window can't post the message to the parent window and just closes. Then the parent window wrongly assumes the popup was closed prematurely and handles it as aborted consent. I'll investigate.

Reply:
Same behavior with FF and Chrome.

Reply:
yes

Reply:
was a caching issue on the oauth consent window